### PR TITLE
feat: Implement `utils.parse_bool`

### PR DESCRIPTION
**Pull request by sveneberth**
Redo https://github.com/phorward/viur-core/pull/5

### CODE REVIEW

**Review by phorward — APPROVED** [review]

**Review by XeoN-GHMB — APPROVED** [review]

File: src/viur/core/bones/boolean.py [hunk 1]
```diff
@@ -45,7 +46,7 @@ def __init__(
             raise ValueError("BooleanBone cannot be multiple")
 
     def singleValueFromClient(self, value, skel, bone_name, client_data):
-        return str(value).strip().lower() in conf["viur.bone.boolean.str2true"], None
+        return parse_bool(value, conf["viur.bone.boolean.str2true"]), None
```
Comment (ArneGudermann) [inline src/viur/core/bones/boolean.py:49]:
```suggestion
        return parse_bool(value), None
```

File: src/viur/core/bones/boolean.py [hunk 2]
```diff
@@ -78,7 +79,7 @@ def refresh(self, skel: 'viur.core.skeleton.SkeletonInstance', boneName: str) ->
             :param name: The property-name this bone has in its Skeleton (not the description!)
         """
         if not isinstance(skel[boneName], bool):
-            skel[boneName] = str(skel[boneName]).strip().lower() in conf["viur.bone.boolean.str2true"]
+            skel[boneName] = parse_bool(skel[boneName], conf["viur.bone.boolean.str2true"])
```
Comment (ArneGudermann) [inline src/viur/core/bones/boolean.py:82]:
```suggestion
            skel[boneName] = parse_bool(skel[boneName])
```

File: src/viur/core/bones/boolean.py [hunk 3]
```diff
@@ -100,7 +101,7 @@ def buildDBFilter(
         :rtype: google.cloud.ndb.query.Query
         """
         if name in rawFilter:
-            val = str(rawFilter[name]).strip().lower() in conf["viur.bone.boolean.str2true"]
+            val = parse_bool(rawFilter[name], conf["viur.bone.boolean.str2true"])
```
Comment (ArneGudermann) [inline src/viur/core/bones/boolean.py:104]:
```suggestion
            val = parse_bool(rawFilter[name])
```

**Review by ArneGudermann — CHANGES_REQUESTED** [review]

**Comment by ArneGudermann** [comment]
Ohh ok I was too late. :smile:

**Pull request merged**